_includes/tutorials/aggregating-count/flinksql/code/tutorial-steps/dev/validate-movie-sales-by-title.sh

```diff
@@ -1,9 +1,8 @@
-docker exec -e SCHEMA_REGISTRY_LOG4J_OPTS=" " -it schema-registry /usr/bin/kafka-avro-console-consumer \
-  --topic movie-ticket-sales-by-title \
+docker exec -it confluent-cli confluent kafka topic consume movie-ticket-sales-by-title \
+  --bootstrap broker:9092 \
+  --protocol PLAINTEXT \
   --from-beginning \
-  --max-messages 10 \
-  --timeout-ms 10000 \
-  --bootstrap-server broker:9092 \
-  --property key.deserializer=org.apache.kafka.common.serialization.StringDeserializer \
-  --property print.key=true \
-  --property key.separator="-"
+  --print-key \
```

Comment:
We'll need to set max messages with --config max-messages=10.
Not having one will break the test for this tutorial. A harness runner executes each step of the tutorial, and we need the consumer to exit after it consumes the expected number of records; otherwise, it will hang and test times out.